yarn.lock

```diff
@@ -4760,6 +4801,17 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@napi-rs/wasm-runtime@npm:^0.2.10":
```

Comment:
what is this? which package depends on it?

Reply:
As above - I accidentally introduced this as an indirect dep of `eslint-import-resolver-typescript` which is unused now - removed it all in 28b1807e9cad6d41706b639b40ba03270cdf6cdc.